### DISCUSSION TITLE
Using build-in compression winapi

### DISCUSSION THREAD

**Discussion by 0x4v0c4d0** [kareldonk/QuantumGate#16] [Ideas]
What you think about using [build-in compression winapi](https://docs.microsoft.com/en-us/windows/win32/api/_cmpapi/) methods instead of 2 external libs: zlib and zstandart?

There is [list of supported compression protocols](https://docs.microsoft.com/en-us/windows/win32/api/compressapi/nf-compressapi-createcompressor):
- MSZIP compression algorithm
- XPRESS compression algorithm 
- XPRESS compression algorithm with Huffman encoding 
- LZMS compression algorithm

Using of build-in winapi is will greatly reduce the weight of executable files, make it easier to build the project and get rid of 2 dependencies.

while the main feature - compression will remain

**Comment by kareldonk** [comment]
The reason why I chose zlib and zstandard is because I want to support Linux in the future and would like to have any algorithms used on Windows to be available there as well.
